Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM alpine:3.18
+
+
+RUN apk add --no-cache ca-certificates
+RUN apk upgrade --update-cache --available
+RUN adduser --disabled-password --no-create-home --uid=1983 spacelift
```

Comment:
Since we're creating a user for this image, should we use that user? Here's what we do in the runner-terraform image: https://github.com/spacelift-io/runner-terraform/blob/224cd1b25829a2fde73ad4a86d0b990ced2640db/Dockerfile#L64